src/amber/server/env.cr

```diff
@@ -0,0 +1,27 @@
+module Amber
+  module Env
+    ENVIRONMENTS = %w(development test production)
+
+    def self.in?(environments : Array(String | Symbol))
+      (ENVIRONMENTS.map &.to_s.downcase).includes? current
+    end
+
+    def self.is?(environment : String | Symbol)
+      current == environment.to_s.downcase
+    end
+
+    def self.current
+      Settings.env.not_nil!.downcase
```

Comment:
Store env here is just duplicating values already stored in ENV["AMBER_ENV"]. Is this necessary? I would rather `current` just wrap `ENV["AMBER_ENV"]` and return it's values.

Reply:
I would trust the Settings.env more than the ENV["AMBER_ENV"] since the latter can be modified outside of the APP after it has been started. What do you think?

Reply:
ENV["AMBER_ENV"] is defined by the env var at runtime and can't be changed outside of the app. Since the env var is where this value originates I forsee some possible issues where they could get out of sync and cause confusion if they're not pointing t

Reply:
We should be fine using `Settings.env` since its pointing to `ENV["AMBER_ENV"]`. See https://github.com/amberframework/amber/pull/302/files#diff-b8648cef4f24d74b1b3b0cee94f1c9deR15

Reply:
Ok. I'm ok with that then.